### ISSUE TITLE
!!!ATTENTION: STOP LEAVING FEEDBACK OR BEGGING HERE!!!

### ISSUE THREAD

**Issue by KamikazePlatypus** [nedwill/soundhax#41]
This is a place for reporting issues, NOT for leaving feedback or begging for the devs to work faster. While the exploit not working in a certain region or on a certain console is technically a bug, it is expected and we don't need 5 issues of people crying that it doesn't work. It will come to your region in time. It's counter-productive and annoying to get the same people begging for a release. Nedwill and other community members are doing this on their own free time and have NO OBLIGATION to release anything. Please, stop spamming the issue page, and more importantly, BE PATIENT. /endrant

(inb4 someone points out that I spammed the page to stop people from spamming the page)

**Comment by JesseBessem** [comment]
well this is not an issue really. since i'm not reporting one also. ill just mention this.
@nedwill I've tested soundhax on N3DS 11.2 and it works like a charm :)

**Comment by jaxrogers2** [comment]
Yeah, I think this is pretty warranted.
Plus, as long as the bug exists in all regions of the sound app (which it already is in 4 out of the 6, so probably so) this will be the first homebrew exploit to work in all regions. However since CHN+TWN don't have hosted Homebrew Launcher payloads that might be difficult to support.

**Comment by KamikazePlatypus** [comment]
@BNTK I felt that this was warranted given the number of sheer repetitive issues.

**Comment by iggyvolz** [comment]
You can make a .github/CONTRIBUTING.md file that will be linked to in the header whenever you create a new issue, and/or a .github/ISSUE_TEMPLATE.md or .github/PULL_REQUEST_TEMPLATE.md file to auto-fill the Issue or Pull Request fields (example: https://github.com/Homebrew/legacy-homebrew/issues/new)

**Comment by PCKid11** [comment]
I wanted to confirm that the NEW-EUR payload works correctly. Thanks! Been waiting on a free entrypoint since Ironhax got patched 😄

**Comment by maorninja** [comment]
While we don't need a backseat moderator, I agree with what you said.

Please post this also at the fasthax repo.
